### DISCUSSION TITLE
Adding Reasons for Hosting to Arrangements

### DISCUSSION THREAD

**Discussion by AGNMM18** [CareTogether/CareTogetherCMS#628] [Ideas]
This is a note to make sure reasons for hosting are added to the arrangement level in the CMS.

**Comment by LarsKemmann** [comment]
This is done! 🎉 Organizations can customize their list of reasons that they want to use, and the reasons also show up in the reports.

**Comment by AGNMM18** [comment]
Yay!  Thank you!


Ashley Grimes

Operations Manager
New Mercies Ministries



M 260-246-0399

E   ***@***.******@***.***>

W www.new-mercies.org<http://www.new-mercies.org/>



________________________________
